e2e/README.md

```diff
@@ -7,27 +7,44 @@
 ##### Clone github project
 - `git clone git@github.com:all-of-us/workbench.git`
 
-##### Set up login credential
-- copy `.env.sample` file, save new file as `.env` (do not commit `.env` file to version control).
-- edit new `.env` file to provide workbench login credential.
-  ** note: we have a 2FA-bypassed test user created to run tests in test env. if you don't know it, ask others for test user credential.
-
-##### install node libraries
+##### Install node libraries
```

Comment:
Above, where you don't have a space after `####`, it's not rendering correctly.
![Screen Shot 2020-05-19 at 12 14 37 PM](https://user-images.githubusercontent.com/53479492/82351289-6549cb80-99ca-11ea-8a7d-5d6a86bf3027.png)

Reply:
fixed